src/projectscene/view/trackspanel/trackslistmodel.cpp

```diff
@@ -172,17 +172,19 @@ QItemSelectionModel* TracksListModel::selectionModel() const
     return m_selectionModel;
 }
 
-void TracksListModel::selectRow(int rowIndex)
+void TracksListModel::selectRow(int row)
 {
+    if (row >= rowCount()) {
+        return;
+    }
+
     if (m_audioDataSelected) {
-        selectionController()->resetSelectedTracks();
+        selectionController()->resetDataSelection();
         clearSelection();
         m_audioDataSelected = false;
     }
 
-    selectionController()->resetDataSelection();
-    QModelIndex modelIndex = index(rowIndex);
```

Comment:
These two lines:
```c++
QModelIndex modelIndex = index(rowIndex);
m_selectionModel->select(modelIndex);
``` 
need to stay, otherwise you'll not be able to select multiple tracks using \"ctrl/shift + track header click\"
(because ItemMultiSelectionModel is not updated)

Reply:
Fixed